example-monorepos/blank/apps/expo/package.json

```diff
@@ -17,7 +17,6 @@
     "@babel/core": "^7.12.9",
     "@types/react": "~17.0.21",
     "@types/react-native": "~0.64.12",
```

Comment:
Expo Docs states that we need to add a property to our package.json called: 
```
 "main": "index.js",
```
Expo Docs: https://docs.expo.dev/guides/monorepos/#change-default-entrypoint

We have `index.js` file in Solito, but it's not being referenced on our package.json. What about add it too?

Reply:
we could pull this into a separate PR if it’s needed (is it?)

Reply:
@nandorojo sure! I'll open a new one with these changes.